ci-operator/config/windup/windup-ui-tests/OWNERS

```diff
@@ -0,0 +1,17 @@
+approvers:
```

Comment:
We need to add some MTR product qe members to this OWNERS file.

Also I think we could probably remove vi-patel, and minlei98

Reply:
I will remove vi-patel and minlei98 in an upcoming commit. I have reached out to PQE about being reviewers/approvers for the MTR scenario. They would like Shveta and Sachin to be on the list. I will send them links to get thier GitHub accounts set up before I add them.

Reply:
Shveta and Sachin have been included as owners